Rivet/Functions/Invoke-Rivet.ps1

```diff
@@ -104,8 +107,14 @@ function Invoke-Rivet
         $Checkpoint,
 
         [Parameter(ParameterSetName='Checkpoint')]
+        [String]
         # The output path for the schema.ps1 file that will be generated when using the -Checkpoint switch. If not provided the path will default to the same directory as the `rivet.json` file.
-        [String] $CheckpointOutputPath
+        $CheckpointOutputPath,
+
+        [Parameter(ParameterSetName='Initialize')]
+        [Switch]
+        # Initializes a database for a migration by applying contents of the schema.ps1 file located at the same directory as the rivet.json file.
+        $Initialize

```

Comment:
Initialization should happen automatically. The schema.ps1 file should be saved to the `Migrations` directory of each database.

It will happen automatically if the `Get-MigrationFile` function is updated to handle the special 'schema.ps1' file. If `Get-MigrationFile` encounters/finds/detects a schema.ps1 file, it should return it with a "special" ID of `10000000000` (basically midnight at year one, month zero, day zero). This migration will then be applied automatically when users push.

When popping, however, this migration should *never* be run/popped. Once pushed, it is immutable.